syncd/NotificationProcessor.cpp

```diff
@@ -403,9 +403,13 @@ void NotificationProcessor::process_on_port_state_change(
         if (false == m_translator->tryTranslateRidToVid(oper_stat->port_id, oper_stat->port_id))
         {
             oper_stat->port_id = SAI_NULL_OBJECT_ID;
+
+            SWSS_LOG_WARN("Port RID transalted to null VID!!!");
```

Comment:
please also log exact RID value, it cannot be oper_stat->port_id, since that is overrided, you need to introduce new variable